File: src/vel_pkg/scripts/vel_node.py
#! /usr/bin/env python3
#coding=utf-8
import rospy
from geometry_msgs.msg import Twist

if __name__=="__main__":
    rospy.init_node("vel_node")
    pub=rospy.Publisher("/cmd_vel",Twist,queue_size=10)
    vel_rate=rospy.Rate(30)
    msg=Twist()

    msg.linear.x=0.1
    msg.linear.y=0
    msg.linear.z=0
    msg.angular.z=0.6
    msg.angular.y=0
    msg.angular.x=0

    while not rospy.is_shutdown():
        pub.publish(msg)
        vel_rate.sleep()

File: src/vel_pkg/src/vel_node.cpp
#include<ros/ros.h>
#include<geometry_msgs/Twist.h>

using geometry_msgs::Twist;
int main(int argc,char** argv)
{
    ros::init(argc,argv,"vel_node");
    ros::NodeHandle nh;
    ros::Publisher pub=nh.advertise<Twist>("/cmd_vel",100);

    Twist vel_msgs;
    vel_msgs.linear.x=0;
    vel_msgs.linear.y=0;
    vel_msgs.linear.z=0;
    vel_msgs.angular.x=0;
    vel_msgs.angular.z=1;
    vel_msgs.angular.y=0;

    ros::Rate vel_rate(30);
    while(ros::ok())
    {
        pub.publish(vel_msgs);
        vel_rate.sleep();
    }
    return 0;
}